tutorials/arvo/claz.md

```diff
@@ -0,0 +1,76 @@
+
+Once you have made these changes be sure to run `|commit %home` to get them into your ship.
+
+Next run `|start %claz` after which you'll be able to generate transactions.
+
+```
+:claz [%generate %/transaction/eth-txs %mainnet author %single %spawn ~ship to]
+```
+
+Here we have an example of generating a transaction to spawn a ship. Something to note is that for this to be successful you'll need to actually have the right to spawn the ship in question. Let's break down the command.
+
+
+`%generate` is how all commands to `claz` are going to start.
+
+`%/transaction/eth-txs` is simply the path for the output file
+
+`%mainnet` specifies which network this is going to sent out on.
+
+`author` here is an etherium address that is issuing the transaction. Remember that it will need to be written all in lower case, dotted notation, e.g.  0x3b17.d097.d9dd.711e.4ef8.517a.bbf1.8b2b.a643.81fe
+
+`%single` specifies what kind of batch that will be created. If you are going to do multiple transactions you will need to either set this to `%more` and create a list of batches or upload them before generating the next one. The reason this ship needs to have internet access is because it needs to look up the nonce for the specified address.
+
+`%spawn` is the command we want to issue. After this what the arguments are will vary based on the command.
+
+`~ship` will be the ship to be spawned.
+
+`to` is the etherium address to spawn the ship to.
+
+You will want to read `sur/claz.hoon` which you can find [here](https://github.com/urbit/urbit/blob/master/pkg/arvo/sur/claz.hoon).
+This will contain more details about the various kinds of transactions you can perform particularly `++call`
+
+After running this command you should find a csv file at the specified path. You can examine this file to learn how they are constructed but it's not important for this guide.
+
+This file can now be signed and uploaded. At this point, you can transfer this file to an airgapped machine which has your keys if you want. If you choose to do that you will need to download the urbit binary as well as the most recent pill file or a copy of the fakezod pier you have already booted.
+
+For this guide, we'll continue on the same ship.
+
+Run `|start %eth-sender` and then you can use it to sign the transaction file.
+
+```
+:eth-sender [%sign %/transaction-signed/txt %/transaction/eth-txs %/mykey/txt ~[80]]
+```
+
+Let's again break this down.
+
+`%sign` is the action we are going to perform here.
```

Comment:
"eth-sender can also be used to submit batches of transactions to the network in various ways, but details on that are outside of scope here." or similar